clang/lib/Headers/emmintrin.h

```diff
@@ -1405,11 +1413,12 @@ static __inline__ __m128d __DEFAULT_FN_ATTRS _mm_cvtss_sd(__m128d __a,
 
 /// Converts the two double-precision floating-point elements of a
 ///    128-bit vector of [2 x double] into two signed 32-bit integer values,
-///    returned in the lower 64 bits of a 128-bit vector of [4 x i32].
+///    returned in the lower 64 bits of a 128-bit vector of [4 x i32],
+///    truncating inexact results.
```

Comment:
The last sentence is not necessary and has ambiguity in truncating the fractional part or truncating higher bits to 32-bit.

Reply:
Truncation is what distinguishes this from `_mm_cvtpd_epi32` so I think it is necessary? That information used to be in the second paragraph, I shortened it and moved it up to the first paragraph, to make that difference more obvious.
Behavior on overflow was added to the second paragraph.

Do you have a different suggestion for how to organize this information?

Reply:
Oh, I took that simply as wrong infromation.

> Do you have a different suggestion for how to organize this information?

Not ideal, something like

```
/// Converts the two double-precision floating-point elements of a
///    128-bit vector of [2 x double] into two signed truncated
///    (rounded towards zero) 32-bit integer values,returned in the
///    most negative integer.
```